### PR TITLE
Prepare for 1.2.0-rc3 release

### PR DESCRIPTION
**Pull request by tgraf**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5184)

### CODE REVIEW

File: examples/kubernetes/1.11/cilium-crio-ds.yaml [hunk 1]
```diff
@@ -221,8 +221,7 @@ spec:
             defaultMode: 420
             optional: true
             secretName: cilium-clustermesh
-      restartPolicy: Always
-      priorityClassName: system-node-critical
+      restartPolicy: Alwaysn      priorityClassName: system-node-critical
```
Comment (joestringer) [inline examples/kubernetes/1.11/cilium-crio-ds.yaml]: What's going on here?

**Review by joestringer — CHANGES_REQUESTED** [review]
Looks like something is going wrong on the `DaemonSet` generation.

File: examples/kubernetes/1.11/cilium-crio-ds.yaml (hunk 1 above)
Comment (ianvernon) [reply examples/kubernetes/1.11/cilium-crio-ds.yaml]: I have had this happen when i ran the commands to generate the `DaemonSet` files on Mac. It needs to be done on Linux :(

Comment (tgraf) [reply examples/kubernetes/1.11/cilium-crio-ds.yaml]: Fixed

**Comment by tgraf** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**